mousestyles/classification/tests/test_classification.py

```diff
@@ -1,5 +1,39 @@
-from mousestyles import classification
+from mousestyles import data
+from mousestyles.classification import classification
 
+def test_prep_data():
+    # check prep_data() returns approriate lists for classification
+    df = data.load_mouseday_features()
+    strain = df['strain']
+    features = df.iloc[:, 3:]
+    train_y, train_x, test_y, test_x = prep_data(strain, features)
+    assert train_y.shape == (1440, )
+    assert train_x.shape == (1440, 99)
+    assert test_y.shape == (481, )
+    assert test_x.shape == (481, 99)
 
-def test_classification():
-    assert classification.dummy()
+
+def test_RandomForest():
+    # check RandomForest() returns approriate data frame with 
+    # prediction labels and true labels
+    df = data.load_mouseday_features()
+    strain = df['strain']
+    features = df.iloc[:, 3:]
+    result = RandomForest(strain, features)
+    assert result.shape == (481, 2)
+    assert all(result.iloc[:,0] >= 0) & all(summary.iloc[:,0] <= 15)
+    assert all(result.iloc[:,1] >= 0) & all(summary.iloc[:,1] <= 15)
```

Comment:
Here is a bug, it's suppose to be:
`assert all(result.iloc[:,0] >= 0) & all(result.iloc[:,0] <= 15)`
`assert all(result.iloc[:,1] >= 0) & all(result.iloc[:,1] <= 15)`